addons/website/views/snippets/s_cta_badge.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template name="CTA Badge" id="s_cta_badge">
+    <span class="s_cta_badge o_animable" data-name="CTA Badge">
+        <span class="s_cta_badge_container border bg-o-color-4" style="border-width: 1px;">
+            <i class="fa fa-1x fa-fw fa-info-circle o_not-animable"/>What do you want to promote ? <a href="#">See more <span class="fa fa-long-arrow-right" contenteditable="false"/></a>
```

Comment:
Is there a reason for the double container? 

Can't we go:
```xml
<template>
    <span>
        <a/>
    </span>
<template>
```
rather then:
```xml
<template>
    <span>
        <span>
            <a/>
        <span/>
    </span>
<template>
```
?

Reply:
The reason is that we need to target a child element to enable the border/color/shadow options (see [L485](https://github.com/odoo/odoo/blob/master/addons/website/views/snippets/snippets.xml#L485)).

I know this structure looks weird but I'm not sure there is a better way (not really an improvement but perhaps we could use a div instead of the first span, making the cta badga container full-width.

Reply:
You're not obliged to use that selector, you can create your own in the snippet's xml file, eg:

```xml

<div data-selector=".s_cta_badge">
    <we-colorpicker string="Colors"
        data-select-style="true"
        data-css-property="background-color"
        data-color-prefix="bg-"/>
    <t t-call="website.snippet_options_border_widgets">
        <t t-set="so_rounded_no_dependencies" t-value="True"/>
    </t>
    <t t-call="website.snippet_options_shadow_widgets"/>
</div>
```

😉